packages/types/src/tx/schema/transfer.ts

```diff
@@ -8,6 +8,7 @@ export class TransferMsgValue {
   source: string;
   target: string;
   token: string;
+  public_key?: string;
```

Comment:
Where does this get used? I couldn't see anywhere but I might just be missing it.

Reply:
https://github.com/anoma/namada-interface/blob/873cd3181a8ede971dd54f850102515fbdd0a172/packages/shared/lib/src/sdk/tx.rs#L276-L316

It gets used here - when this msg is deserialized, we can use the public key (if provided) to send as the `verification_key`

Reply:
At first I thought this was the Tx props - but you are correct, `public_key` got moved to Tx and out of transfer, and I just forgot to remove it!